docs/content/blog/using-docker-in-bundles.md

```diff
@@ -77,138 +76,100 @@ required:
 
 ### Install Docker
 
-We can install the Docker CLI, or whatever is needed, into the
-bundle using a [custom Dockerfile](https://porter.sh/custom-dockerfile/).
+You can include the Docker CLI in your bundle by using the [docker mixin], or
+if you are using Docker Compose, use the [docker-compose mixin].
 
-In our [Dockerfile.tmpl](https://porter.sh/src/examples/docker/Dockerfile.tmpl)
-below, I am installing the Docker CLI and copying my files into the bundle:
-
-```dockerfile
-FROM debian:stretch-slim
-
-ARG BUNDLE_DIR
-RUN apt-get update && apt-get install -y curl ca-certificates
-
-ARG DOCKER_VERSION=19.03.8
-RUN curl -o docker.tgz https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKER_VERSION}.tgz && \
-    tar -xvf docker.tgz && \
-    mv docker/docker /usr/bin/docker && \
-    chmod +x /usr/bin/docker && \
-    rm docker.tgz
-
-# Use the BUNDLE_DIR build argument to copy files into the bundle
-COPY . $BUNDLE_DIR
+```yaml
+mixins:
+  - docker
+  - docker-compose
```

Comment:
This makes it look like you need to include both mixins.

Reply:
Oh thanks! Yeah I was trying to talk about both mixins at once and I think that's even more confusing. Let me just focus on the docker mixin.